### DISCUSSION TITLE
CALL db.alterLabelAddFields 接口报错

### DISCUSSION THREAD

**Discussion by HTXone** [TuGraph-family/tugraph-db#693] [Q&A]
根据官方文档执行example报错（已创建vertex）
![image](https://github.com/user-attachments/assets/242e1893-1ac6-4c8b-86c4-e074b448f3f8)

报错：
Failed to get response from server after retrying: Error returned from server, code=[500], msg=[CypherException: visit(...) failed at src/cypher/arithmetic/ast_expr_evaluator.cpp:480]

版本：
docker: tugraph/tugraph-runtime-centos7:latest

![image](https://github.com/user-attachments/assets/a32626a5-4449-48f6-bdda-2c69db418ed9)

**Comment by zhangwh807** [comment]
如上述DATE等数据类型加上引号
